README.md

```diff
@@ -26,8 +26,47 @@ the right to make this assignment.
 
 -----
 
-Hasjob runs on [Python][] with the [Flask][] microframework. You will need
-to install all the requirements listed in `requirements.txt` using
+### Configuration
+
+Copy `settings-sample.py` to `settings.py`.
```

Comment:
We ship a `settings.py` in the repo, so this should be `development.py`. (However, the Docker deployment has a custom `development.py` as well. @karthikb351?)